src/Microsoft.AspNetCore.HttpOverrides/ForwardedHeadersOptions.cs

```diff
@@ -10,6 +10,38 @@ namespace Microsoft.AspNetCore.Builder
     public class ForwardedHeadersOptions
     {
         /// <summary>
+        /// X-Forwarded-For
+        /// </summary>
+        public const string DefaultForwardedForHeaderName = "X-Forwarded-For";
+
+        /// <summary>
+        /// X-Forwarded-Host
+        /// </summary>
+        public const string DefaultForwardedHostHeaderName = "X-Forwarded-Host";
+
+        /// <summary>
+        /// X-Forwarded-Proto
+        /// </summary>
+        public const string DefaultForwardedProtoHeaderName = "X-Forwarded-Proto";
+
+        /// <summary>
+        /// Use this header name instead of X-Forwarded-For
+        /// </summary>
+        public string ForwardedForHeaderName { get; set; } = DefaultForwardedForHeaderName;
```

Comment:
In the ForwardedHeadersMiddleware constructor validate that these are not null or empty